content/ops/troubleshooting-bosh.md

```diff
@@ -5,49 +5,72 @@ menu:
 title: Troubleshooting Bosh
 ---
 
-## Accessing Jump Boxes
+## Accessing jumpboxes
 
-If you're going to be accessing Bosh, you will need to intercept a jumpbox via Concourse.
+If you're going to be accessing Bosh, you will need to intercept a jumpbox via
+Concourse.
 
 ### Pre-flight checklist:
+
 1. Need to have access to Concourse for cloud.gov (e.g., https://ci.cloud.gov)
 1. Need to have `fly` installed locally
```

Comment:
I'm going to be pointing to [the official documentation](https://github.com/concourse/fly#installing-from-the-concourse-ui-for-project-development) along with updating the description on our own for added clarity.